controller/iteration_blackbox_test.go

```diff
@@ -38,6 +42,7 @@ type TestIterationREST struct {
 	gormtestsupport.DBTestSuite
 	db      *gormapplication.GormDB
 	testDir string
+	policy  *auth.KeycloakPolicy
 }
 
 func TestRunIterationREST(t *testing.T) {
```

Comment:
Just for thought: we should rename the main test func to something like `TestIterationController` instead, along with a `TestIterationControllerSuite` struct, IMO... (change is not required for this PR)

Reply:
sure, I agree, I will keep it in OSIO issue as of now and once I merge these PRs(https://github.com/fabric8-services/fabric8-wit/pull/1783, https://github.com/fabric8-services/fabric8-wit/pull/1770), I will create new PR for this refactoring.

Reply:
ok